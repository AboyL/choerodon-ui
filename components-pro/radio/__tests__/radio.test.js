import React from 'react';
import { render } from 'enzyme';
import Radio from '..';
import focusTest from '../../../tests/shared/focusTest';

describe('Radio-pro', () => {
  focusTest(Radio);

  it('should render correctly', () => {
    const wrapper = render(<Radio>Test</Radio>);
    expect(wrapper).toMatchSnapshot();
  });
});
