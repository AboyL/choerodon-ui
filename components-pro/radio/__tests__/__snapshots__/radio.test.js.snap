// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio-pro should render correctly 1`] = `
Array [
  <label
    class="c7n-pro-radio-wrapper"
  >
    <input
      class="c7n-pro-radio"
      type="radio"
      value="on"
    />
    <span
      class="c7n-pro-radio-inner"
    />
    <span
      class="c7n-pro-radio-label"
    >
      Test
    </span>
  </label>,
  <span />,
]
`;
